modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -30,7 +31,7 @@ image:invite_user.png[]
 ====
 Capabilities of users with the role "Metrics Integration Reader" are limited to fetching the metrics and getting a read-only view of the tenant.
 ====
-. In https://console.neo4j.io/#account["Account Details"^], create new Aura API credentials. Save client secret. +
+. Ensure you are logged in to Aura as the user selected in the previous step. In https://console.neo4j.io/#account["Account Details"^], create new Aura API credentials. Save client secret. +
```

Comment:
Would it be a good idea to add a note in CMI view in Console UI to make sure when the users copy the URLs or the prom config, they would be notified of the requirements.

Reply:
Yes, nice idea, [carded](https://trello.com/c/ev160pfn/2268-add-a-note-in-cmi-view-in-console-ui-to-make-sure-when-the-users-copy-the-urls-or-the-prom-config-they-would-be-notified-of-the) it